Major doc update para seller behavior (#1258)
mercadopago/devsite-docs: guides/resources/devpanel/devpanel.en.md

# Developer Panel
The [Developer Panel](https://mercadopago[FAKER][URL][DOMAIN]/developers/panel) is your application management environment. It is automatically created with a user ID for you when you open a Mercado Pago account. 

> Applications are nothing more than the different integrations one or more stores have. You can create an application for each solution implemented to have everything organized and maintain control to facilitate your management. 

Reviewer: ```suggestion
Applications are nothing more than different integrations that one or more stores have. You can create an application for each solution implemented, so you have everything organized and maintain control to facilitate your management. 
```

> WARNING 
> 
> Warning
> 
> If you are not a developer, we recommend that you integrate Mercado Pago to your store through [plugins and platforms](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/plugins) and retrieve your credentials through your Mercado Pago account in [Your business > Settings > Management and Administration > Credentials](https://www.mercadopago[FAKER][URL][DOMAIN]/settings/account/credentials). Read [Credentials](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/credentials/credentials) for more information.

## Create application
To create an application, follow the steps below:

1. Select "New application" or "Create your first application".
2. Name your application.
3. Choose a solution to be integrated.
4. Accept our Terms and Conditions.
5. Create the application by clicking on the button "Create application".

> To every application created, a new card is also automatically created on the Developer Panel's homepage, containing the application's name, description, and ID.

Reviewer: ```suggestion
To every application created, a new card is automatically created on the Developer Panel's homepage, containing the application's name, description, and ID.
```

## Application details

> By clicking on an application card, you will have access to application details, including:

Reviewer: ```suggestion
By clicking on an application card, you will have access to it's details, including:
```
